ocaml/sdk-gen/go/test_data/record_convert.go

```diff
@@ -0,0 +1,35 @@
+func serializeVBDRecord(context string, record VBDRecord) (rpcStruct map[string]interface{}, err error) {
+	rpcStruct = map[string]interface{}{}
+	rpcStruct["uuid"], err = serializeString(fmt.Sprintf("%s.%s", context, "uuid"), record.UUID)
+	if err != nil {
+		return
+	}
+	rpcStruct["allowed_operations"], err = serializeEnumVbdOperationsSet(fmt.Sprintf("%s.%s", context, "allowed_operations"), record.AllowedOperations)
+	if err != nil {
+		return
+	}
+	return
+}
+
+func deserializeVBDRecord(context string, input interface{}) (record VBDRecord, err error) {
+	rpcStruct, ok := input.(map[string]interface{})
+	if !ok {
+		err = fmt.Errorf("failed to parse XenAPI response: expected Go type %s at %s but got Go type %s with value %v", "map[string]interface{}", context, reflect.TypeOf(input), input)
+		return
+	}
+	uuidValue, ok := rpcStruct["uuid"]
```

Comment:
The test result here is different with generated Go method:
```go
uUIDValue, ok := rpcStruct["uuid"]
	if ok && uUIDValue != nil {
		record.UUID, err = deserializeString(fmt.Sprintf("%s.%s", context, "uuid"), uUIDValue)
		if err != nil {
			return
		}
	}
``` 
uuidValue , uUIDValue , why ?

Reply:
`uuidValue  `is a sample data for testing template. `uUIDValue` is real generated data. It's a little weird now. I'll fixed this later.